Tests/pens/ttGlyphPen_test.py

```diff
@@ -290,6 +290,310 @@ def test_scaled_component_bounds(self):
         self.assertGlyphBoundsEqual(uni0302_uni0300, (-278, 745, 148, 1025))
 
 
+class TTGlyphPointPenTest:
+    def runEndToEnd(self, filename):
```

Comment:
this `runEndToEnd` is basically identical to the one in `runEndToEnd` (except for the pen class) maybe have a base class that defines that and have both subclass from it, with the pen class defined as a class attribute